### ISSUE TITLE
Use a computer vision library

### ISSUE THREAD

**Issue by Vejron** [fighterfish/android_tracking_app#1]
I think "we" should look at the possibility of using BoofCV. It could really speed up the development process and make the app more performant. Never used it myself but it looks a lot like openCV and claims to be compatible with processing and android.

**Comment by fighterfish** [comment]
Yeah, I have compiled some sketches using boofcv , it has blob detection
and all that but I hit my limit of capabilities and couldn't figure out how
to do motion tracking and returning the upper limit ie a persons head to
determine where they are in the frame. So I opted to track a color instead.
Like in a headband.
